lib/portlayer/network/context.go

```diff
@@ -1258,6 +1258,35 @@ func (c *Context) DeleteScope(ctx context.Context, name string) error {
 		return fmt.Errorf("%s has active endpoints", s.Name())
 	}
 
+	var allzeros, allones net.IP
+	if !ip.IsUnspecifiedSubnet(s.subnet) {
+		allzeros = ip.AllZerosAddr(s.subnet)
+		allones = ip.AllOnesAddr(s.subnet)
+	}
+
+	for _, p := range s.spaces {
+		for _, i := range append(s.dns, s.gateway, allzeros, allones) {
+			if !ip.IsUnspecifiedIP(i) {
+				p.ReleaseIP4(i)
+			}
+		}
+
+		if p.Parent != nil {
+			p.Parent.ReleaseIP4Range(p)
```

Comment:
here has a check for if Parent is nil:
https://github.com/vmware/vic/blob/master/lib/portlayer/network/ipam.go#L334
Does that create problem?